CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # TRILLIAN Changelog
 
+## v1.1.0 - CloudSpanner Improvements & Log Root structure changes etc.
+
+Published 2018-04-17 08:02:50 +0000 UTC
+
+Changes are in progress (e.g. see #1037) to rework the internal signed root format used by the log RPC server to be more useful / interoperable. Currently they are mostly internal API changes to the log and map servers. However, the `signature` and `log_id` fields in SignedLogRoot have been deleted and users must unpack the serialized structure to access these now. This change is not backwards compatible.

```

Comment:
`#1037` isn't autolinking

Reply:
I was hoping to avoid manual edits to the text that comes back from the github api.

Reply:
Ah, OK, no matter then.